package.json

```diff
@@ -182,7 +182,7 @@
     "path": "^0.12.7",
     "path-browserify": "^1.0.1",
     "prettier": "2.7.1",
-    "prisma": "^3.12.0",
+    "prisma": "4",
```

Comment:
This is a duplicate and can be removed since we have prisma in dependencies

Reply:
prisma is for the CLI and @prisma/client is the module where the autogenerated client actually sits. I don't think these are duplicates. see https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client